lib/rules/require-lang-attribute.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+const ERROR_MESSAGE = 'The `<html>` element must have a `lang` attribute';
+
+module.exports = class RequireLangAttribute extends Rule {
+  visitor() {
+    return {
+      ElementNode(node) {
+        if (node.tag === 'html' && !AstNodeInfo.hasAttribute(node, 'lang')) {
```

Comment:
This should probably also check to see that the value isn't a literal empty string.

Reply:
updated to include a check for this (and added a test)